docker/build_base/udfs/Cargo.toml

```diff
@@ -12,6 +12,7 @@ chrono = "0.4"
 serde = "1.0"
 serde_json = "1.0"
 arroyo-types = { path = "/opt/arroyo/src/arroyo-types" }
+regex = "1"
```

Comment:
Can you also add this to the code that generates the whole project in `arroyo-controller/src/compiler.rs`?